IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -2205,6 +2209,8 @@ private void OpenTiffToImageSharp(ReadOnlySpan<byte> bytes)
                 // open a TIFF stored in the stream
                 using (Tiff tif = Tiff.ClientOpen("in-memory", "r", tiffStream, new TiffStream()))
```

Comment:
Can you change the other instances where it's `tif` as well?

Reply:
Thank for noticing this. I already change all `tif` in `AnyBitmap to `tiff`.